modules/olmv1-about-target-versions.adoc

```diff
@@ -3,106 +3,70 @@
 
 .Example CR with a specified channel
 [source,yaml]
 ----
 apiVersion: operators.operatorframework.io/v1alpha1
 kind: Operator
 metadata:
-  name: quay-example
+  name: pipelines-operator
 spec:
-  packageName: quay-operator
-  channel: stable-3.8 <1>
+  packageName: openshift-pipelines-operator-rh
+  channel: latest <1>
 ----
-<1> Installs the latest release published to the specified channel. Updates to the channel are automatically installed.
+<1> Installs the latest release that can be resolved from the specified channel. Updates to the channel are automatically installed.
 
-If you specify the Operator's target version in the CR, {olmv1} installs the specified version. When the target version is specified in the Operator's CR, {olmv1} does not change the target version when updates are published to the catalog.
+If you specify the Operator or extension's target version in the CR, {olmv1} installs the specified version. When the target version is specified in the CR, {olmv1} does not change the target version when updates are published to the catalog.
 
-If you want to update the version of the Operator that is installed on the cluster, you must manually update the Operator's CR. Specifying a Operator's target version pins the Operator's version to the specified release.
+If you want to update the version of the Operator that is installed on the cluster, you must manually edit the Operator's CR. Specifying a Operator's target version pins the Operator's version to the specified release.
 
 .Example CR with the target version specified
 [source,yaml]
 ----
 apiVersion: operators.operatorframework.io/v1alpha1
 kind: Operator
 metadata:
-  name: quay-example
+  name: pipelines-operator
 spec:
-  packageName: quay-operator
-  version: 3.8.12 <1>
+  packageName: openshift-pipelines-operator-rh
+  version: 1.11.1 <1>
 ----
-<1> Specifies the target version. If you want to update the version of the Operator that is installed on the cluster, you must manually update this field the Operator's CR to the desired target version.
-
-If you want to change the installed version of an Operator, edit the Operator's CR to the desired target version.
+<1> Specifies the target version. If you want to update the version of the Operator or extension that is installed, you must manually update this field the CR to the desired target version.
 
-[WARNING]
-====
-In previous versions of OLM, Operator authors could define upgrade edges to prevent you from updating to unsupported versions. In its current state of development, {olmv1} does not enforce upgrade edge definitions. You can specify any version of an Operator, and {olmv1} attempts to apply the update.
-====
+If you want to define update edges for an Operator or extension that follows that semantic versioning (semver) standard, you can specify a version range using a comparison string.
```

Comment:
You're not really `defining update edges` by setting a version range, you're specifying a version range that will be resolved to the highest valid value by the operator controller.